src/index.js

```diff
@@ -154,18 +158,19 @@ const render = async (data) => {
     label.innerHTML = likeTemplate;
 
     const noLikes = document.createElement('span');
-    noLikes.classList.add('no-likes');
+    // noLikes.classList.add('no-likes');
```

Comment:
Greate job so far Lukman 👏

Kindly make sure the commented block of code is deleted in your code base